rubocop/lib/rubocop.yml

```diff
@@ -0,0 +1,335 @@
+require: rubocop-rspec
+
+AllCops:
+  Exclude:
+    - bin/**/*
+
+# LINT
+
+Lint/AmbiguousBlockAssociation:
+  # `scope :name, -> (param) { ... }` is a valid syntax
+  Enabled: false
+
+Lint/AmbiguousOperator:
+  # REVIEW
+  Enabled: true
+
+Lint/AssignmentInCondition:
+  # REVIEW
+  # It allows to use `if (foo = 123)` instead of `if foo = 123`
+  Enabled: true
+
+Lint/NonLocalExitFromIterator:
+  Enabled: false
+
+Lint/HandleExceptions:
+  Enabled: false
+
+Lint/LiteralInCondition:
+  # backtics? seriously?
+  Enabled: false
```

Comment:
Hmm. In my opinion, this is a good cop to keep enabled. It disallows "magical" conditions like `if 20` or `if true` and forces you to write more self-documenting code.